kubernetes/helm_charts/core/taxonomy/values.j2

```diff
@@ -4,6 +4,9 @@ namespace: {{ namespace }}
 imagepullsecrets: {{ imagepullsecrets }}
 dockerhub: {{ dockerhub }}
 
+env:
+  javaoptions: {{content_java_mem_limit|default('-Xmx600m')}}
```

Comment:
Please change this to `taxonomy_java_mem_limit`

Reply:
Oh okay